great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -47,7 +47,42 @@ class ResponsePayload(TypedDict):
 AnyPayload = Union[ResponsePayload, ErrorPayload]
 
 
-def _get_user_friendly_error_message(
+def construct_url(
+    base_url: str,
+    organization_id: str,
+    resource_name: str,
+    id: Optional[str] = None,
+) -> str:
+    url = urljoin(
+        base_url,
+        f"organizations/{organization_id}/{hyphen(resource_name)}",
+    )
+    if id:
+        url = f"{url}/{id}"
+    return url
+
+
+def construct_json_payload(
+    resource_type: str,
+    organization_id: str,
+    attributes_key: str,
+    attributes_value: Any,
+    **kwargs: dict,
+) -> dict:
+    data = {
+        "data": {
+            "type": resource_type,
+            "attributes": {
+                "organization_id": organization_id,
+                attributes_key: attributes_value,
+                **kwargs,
+            },
+        }
+    }
+    return data
+
+
+def get_user_friendly_error_message(
```

Comment:
Why are we making this public?

Reply:
this is now being used in the migrator - happy to refactor later if we deem it appropriate but a public util function seems to make sense here?